### ISSUE TITLE
Bug during character creation

### ISSUE THREAD

**Issue by Nephelos** [chaosvolt/cdda-arcana-mod#189]
DEBUG    : Error: Invalid enum string 'FATIGUE' for '17magic_energy_type'

 FUNCTION : bool main_menu::new_character_tab()
 FILE     : src/main_menu.cpp
 LINE     : 863
 VERSION  : 6616372

(Build 2021 07 04 1015/Version 6616372 (tiles))

**Comment by linonetwo** [comment]
This also blocks save loading

**Comment by linonetwo** [comment]
DEBUG    : Error: Invalid enum string 'FATIGUE' for '17magic_energy_type'

 FUNCTION : bool main_menu::load_character_tab(bool)
 FILE     : src/main_menu.cpp
 LINE     : 1149
 VERSION  : 1ddd145

**Comment by linonetwo** [comment]
caused by this? https://github.com/CleverRaven/Cataclysm-DDA/pull/47852
But this is merged in May, don't know ...

**Comment by chaosvolt** [comment]
Okay, this is weird. I tested this on 0.F stable and couldn't reproduce this. Then I tested this on today's experimental build and it's there. Searched through and found out the cause, an item spell I'd recently added was set to cast from fatigue and I forgot about it when porting that spell over to the DDA version.

Annoying thing is, the PR that removed that feature was merged months ago, so I don't know why it didn't report this bug until after 0.F stable. Did they only removing the code from it afterwards or something?
